### PR TITLE
Using Pipfile instead of setup.py + updated README

### PR DESCRIPTION
**Pull request by idanmiara**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,32 @@
...
+- Install Python 3.8 (i.e. to C:\Python38)
+- Install PyCharm (Community edition is good enough)
+- VCS -> Checkout from version control -> git
+    - **https://github.com/CoronaHakab/CoronaHakabModel.git**
+    - choose directory as you please
+- File -> Settings -> Project Interpreter -> Right click on the cog on the top right -> add
+    - Pipenv environment ->
+        - Base Interpreter Python 3.8 (C:\Python38)
+        - Check install packages from pipfile
+        - Pipenv executable (C:\Python38\Scripts\pipenv.exe)
```
Comment (bentheiii) [inline README.md:13]: pipenv is not installed by default

File: Pipfile [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+yappi = "*"
+pytest = "*"
+tox = "*"
+flake8 = "*"
+isort = "*"
+black = "*"
```
Comment (bentheiii) [inline Pipfile:12]: add swimport

Comment (idanmiara) [reply Pipfile:12]: done

File: README.md (hunk 1 above)
Comment (idanmiara) [reply README.md:13]: done

**Pull request merged**